crates/pgmq/src/lib.rs

```diff
@@ -184,7 +184,7 @@ pub struct Message<T = serde_json::Value> {
 }
 
 /// Main controller for interacting with a queue.
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
What does this do?

Reply:
implements the Clone trait on the struct. So that you can do stuff like:

```rust
let q = PgmQueue();
q.clone()
```